scripts/test_workflow.sh

```diff
@@ -69,7 +69,7 @@ check_ret $? "Test Couler failed"
 ############# Run SQLFLow test with Argo Mode #############
 function test_workflow() {
     # start a SQLFlow MySQL Pod with testdata
-    kubectl run mysql --port 3306 --env="SQLFLOW_MYSQL_HOST=0.0.0.0" --env="SQLFLOW_MYSQL_PORT=3306" --image=sqlflow/sqlflow --command -- bash /start.sh mysql
+    kubectl run mysql --port 3306 --env="SQLFLOW_MYSQL_HOST=0.0.0.0" --env="SQLFLOW_MYSQL_PORT=3306" --image=sqlflow:submitter --command -- bash /start.sh mysql
```

Comment:
Maybe can use the env `SQLFLOW_WORKFLOW_STEP_IMAGE` here which been configurated in 
https://github.com/sql-machine-learning/sqlflow/blob/9d64323bef36fc3bc87c5f51f94917f5c2322ce0/.travis.yml#L67